### ISSUE TITLE
Shortcut for "copy as text path + name"

### ISSUE THREAD

**Issue by danielkrajnik** [cboxdoerfer/fsearch#405]
It would be helpful if there was a shortcut to quickly copy the path and name of a file from the fsearch results.

On KDE's Dolphin Ctrl+Alt+C is assigned for that.

This function already exists in fsearch under right click menu "copy as text" => "path + name". I think that it would streamline workflow a bit more if one could just use a keyboard shortcut.

There wasn't any option to assign custom shortcuts for that in preferences menu.

**Comment by YMan84** [comment]
+1 on this!

In the meantime, one of the recent releases added a keyboard shortcut to open up the context menu, so you can hook into that with an xdotool script. I execute the following script with a keymapper shortcut.

`
# Open context menu and Copy Path + Name as text.
xdotool key shift+F10
sleep .25
xdotool key Down + Down + Down + Down + Down
xdotool key Right
xdotool key Down + Down
xdotool key Enter
`

**Comment by danielkrajnik** [comment]
@YMan84 thanks, but shift+f10 didn't seem to do anything on my machine:

FSearch 0.2alpha (PPA-nightly)
Ubuntu 22.04.1 LTS
WM: KWin
DE: KDE

**Comment by cboxdoerfer** [comment]
In the long term this should be covered by the custom shortcut settings, where you'll be able to assign shortcuts for practically any action in FSearch.

In the meantime I'll think about adding the shortcut manually. Are there any other file managers, other than Dolphin, which have a shortcut for copying the file path as text? So we can pick a sane default.

> but shift+f10 didn't seem to do anything on my machine:

> FSearch 0.2alpha (PPA-nightly)

This sounds like your FSearch package hasn't been updated in a while and might be missing the shift+f10 shortcut. The current dev builds should show:
`FSearch 0.3.alpha0 (PPA-nightly)`

**Comment by danielkrajnik** [comment]
@cboxdoerfer you are right, I have add-apt-repository'd fsearch's ppa again and updated 0.3 version had shift+f10 working. Thanks

Custom shortcuts feature sounds exciting!
